CHANGELOG.md

```diff
@@ -24,6 +24,11 @@ constructor. ([#6240](https://github.com/biocore/scikit-bio/issues/624))
 * Added `Sequence.observed_chars` property for obtaining the set of observed characters in a sequence. ([#1075](https://github.com/biocore/scikit-bio/issues/1075))
 * Added `Sequence.frequencies` method for computing character frequencies in a sequence. ([#1074](https://github.com/biocore/scikit-bio/issues/1074))
 * Added experimental class-method ``Sequence.concat`` which will produce a new sequence from an iterable of existing sequences. Parameters control how positional metadata is propagated during a concatenation.
+* Added phylogenetic diversity metrics, including weighted UniFrac, unweighted UniFrac, and Faith's Phylogenetic Diversity. These are accessible as ``skbio.diversity.beta.unweighted_unifrac``, ``skbio.diversity.beta.weighted_unifrac``, and ````skbio.diversity.alpha.faith_pd``, respectively.
```

Comment:
I like the distinction, thanks. Would this approach mask which citation to
use though?
On Nov 19, 2015 7:18 AM, "Greg Caporaso" <notifications@github.com> wrote:

> In CHANGELOG.md
> <https://github.com/biocore/scikit-bio/pull/1040#discussion_r45352858>:
>
> > @@ -24,6 +24,11 @@ constructor. ([#6240](https://github.com/biocore/scikit-bio/issues/624))
> >  * Added `Sequence.observed_chars` property for obtaining the set of observed characters in a sequence. ([#1075](https://github.com/biocore/scikit-bio/issues/1075))
> >  * Added `Sequence.frequencies` method for computing character frequencies in a sequence. ([#1074](https://github.com/biocore/scikit-bio/issues/1074))
> >  * Added experimental class-method ``Sequence.concat`` which will produce a new sequence from an iterable of existing sequences. Parameters control how positional metadata is propagated during a concatenation.
> > +* Added phylogenetic diversity metrics, including weighted UniFrac, unweighted UniFrac, and Faith's Phylogenetic Diversity. These are accessible as ``skbio.diversity.beta.unweighted_unifrac``, ``skbio.diversity.beta.weighted_unifrac``, and ````skbio.diversity.alpha.faith_pd``, respectively.
>
> I thought about this, but the changelog is for users. From a user's
> perspecitive, UniFrac is a metric, not an algorithm, and the implementation
> doesn't impact the results, so we should just be calling it UniFrac. Note
> this is a different situation than Striped Smith-Waterman, for example,
> which is a different implementation of Smith-Waterman but which can impact
> the results.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/1040/files#r45352858>.
>